### ISSUE TITLE
When initial form data is empty, the widget doesn't create the expected data structure

### ISSUE THREAD

**Issue by ooppsss60** [bhch/django-jsonform#130]
I have this schema: 
`{
    "type": "array",
    "items": {
        "anyOf": [
            {
                "title": "Text",
                "type": "object",
                "keys": {
                    "name": {
                        "type": "string"
                    },
                    "type": {
                        "const": "text",
                    },
                    "value": {
                        "type": "string"
                    }
                }
            },
            {
                "title": "Tags",
                "type": "object",
                "keys": {
                    "name": {
                        "type": "string"
                    },
                    "type": {
                        "const": "tags"
                    },
                    "value": {
                        "type": "array",
                        "items": {
                            "type": "string"
                        },
                        "minItems": 1
                    }
                }
            }
        ]
    },
    "minItems": 1,
    "maxItems": 10
}`

Property 'type' has no value in my form:

<img width="388" alt="image" src="https://github.com/bhch/django-jsonform/assets/40275556/3063aeeb-75a5-4870-96a2-b07852c08685">

This causes an error: 'Some required fields are missing'

**Comment by bhch** [comment]
I just tested your given schema in the [playground](https://bhch.github.io/react-json-form/playground/). It is working as expected.

```json
{
    "type": "array",
    "items": {
        "anyOf": [
            {
                "title": "Text",
                "type": "object",
                "keys": {
                    "name": {"type": "string"},
                    "type": {"const": "text"},
                    "value": {"type": "string"}
                } 
            },
            {
                "title": "Tags",
                "type": "object",
                "keys": {
                    "name": {"type": "string"},
                    "type": {"const": "tags"},
                    "value": {
                        "type": "array",
                        "items": {"type": "string"},
                        "minItems": 1
                    }
                }
            }
        ]
    },
    "minItems": 1,
    "maxItems": 10
}
```

**Comment by ooppsss60** [comment]
In playground, it works perfect, but in Django admin not(
I use python3.9, Django 4.2.7

**Comment by bhch** [comment]
I just tested it in django admin. Working fine for me. 

`const` support was added in version `2.20.0`. I'm guessing you're running an older version?

You can check the version using:

```sh
$ python3 -c "import django_jsonform; print(django_jsonform.__version__)"
```

Also, be sure to refresh your browser cache as it's also possible that the browser is still loading an old cached version of the js.

**Comment by bhch** [comment]
Fixed in [v2.20.2](https://github.com/bhch/django-jsonform/releases/tag/v2.20.2). Please upgrade and test it out.

Thank you for reporting this issue.
